src/containers/Dashboard/index.js

```diff
@@ -59,12 +61,40 @@ class DashboardComponent extends React.Component {
   }
 
   render() {
+    const { profileStrength, history } = this.props;
+
     var settings = {
       dots: true,
       arrows: false
     };
+
+    const profileStrengthBanner = profileStrength < 100 && (
+      <PageBanner wrapClass="pageWrapper-large">
+        <div className={`${styles.profileStrength}`}>
+          <Text typeScale="Small" color="defaultGrey" inline>
```

Comment:
@corwinharrell may want to play with the Text styles used here in general. The background color and the text of this seem not to match best.

Reply:
We have not dived into AA specific discussions yet.

Reply:
@Calpoog similarly to my comment earlier re:color palette divergence, we have may have lost a few accessibility points during the translation from design to dev. I'm with you on the fact that we should try and maintain a minimum of an AA rating. We should refactor our values to make sure they achieve that. Good catch here and it's awesome to see that you're considering things like accessibility 👍